src/screens/deployer/index.tsx

```diff
@@ -0,0 +1,446 @@
+import { useRef, useState } from "react";
```

Comment:
guessing this file is kept for reference but is temporary, right?

Reply:
yes